examples/with-bitcoin/package.json

```diff
@@ -1,18 +1,18 @@
 {
   "name": "@turnkey/example-with-bitcoin",
-  "version": "0.1.0",
+  "version": "0.2.0",
```

Comment:
wonder if we should start maintaining changelogs for examples 🫣

Reply:
Yeah I had the same thought. But...I don't really want to introduce more friction to the process. Most examples don't require changelogs; maybe this one is a bit special!